src/redis/BetterSocialCronQueue.js

```diff
@@ -59,17 +59,17 @@ class BetterSocialCronQueue extends BetterSocialQueue {
   static process(job, done, queueInjection) {
     const {flag} = job.data;
     console.log(`============= DOING CRON JOB (${flag})  =============`);
-    if (flag === 'dailyCredderUpdate') {
-      return credderDailyScoreProcess(job, done, queueInjection?.credderScoreQueue);
-    }
+    // if (flag === 'dailyCredderUpdate') {
+    //   return credderDailyScoreProcess(job, done, queueInjection?.credderScoreQueue);
+    // }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code related to 'dailyCredderUpdate' has been commented out. If Credder functionality is no longer needed, it would be better to remove this code entirely rather than commenting it out. Commented-out code can lead to confusion and clutter in the codebase.

```diff
- // if (flag === 'dailyCredderUpdate') {
- //   return credderDailyScoreProcess(job, done, queueInjection?.credderScoreQueue);
- // }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->